packages/pwa-kit-extension-sdk/README.md

```diff
@@ -107,6 +107,12 @@ Now when the base application is built the import for the `logo.ts` in `extensio
 
 This mechanism is useful when you want to allow for fine grained customization of your application extension. Its the responsibility of the extension developer to ensure their document what files are overridable and what the expected input and output of those files are. For example, if the overridable file is a React component you should document the props that get passed to that component and the expected exports of the file.
 
+#### Forced overrides
+In a case when it is necessary to override some files from an extension but the extension developers do not allow those files to be overridden, you can use `forced_overrides` to bypass the restrictions.
+To do this, create a `.forced_overrides` dot file at the root of your base project and list the files you intend to override. Any files specified in this list will be treated as overridable.
+
+Note: It is advisable to use this function with caution since it could potential break intended behavior of installed extensions.   
```

Comment:
I'd also note that this should be a temporary solution while seeking proper extensibility support.
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/1f2bc449f04c1b102889607310c31eb28d1ab330/packages/template-typescript-minimal/.force_overrides#L5